api/src/main/java/marquez/db/mappers/MapperUtils.java

```diff
@@ -1,15 +1,26 @@
 package marquez.db.mappers;
 
+import static marquez.db.Columns.stringOrNull;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import java.sql.ResultSet;
 import java.sql.ResultSetMetaData;
 import java.sql.SQLException;
 import java.util.HashSet;
+import java.util.Optional;
 import java.util.Set;
+import lombok.NonNull;
 import lombok.extern.slf4j.Slf4j;
+import marquez.common.Utils;
+import marquez.common.models.Facets;
+import marquez.db.Columns;
 
 @Slf4j
-public class MapperUtils {
+public final class MapperUtils {
```

Comment:
I think there's a `@Utils` lombok annotation to generate final classes with private constructors

Reply:
Woah, very cool. Let's use `@UtilityClass`!